app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/itemlisting/VaultItemListingScreen.kt

```diff
@@ -137,6 +144,30 @@ fun VaultItemListingScreen(
             is VaultItemListingEvent.CompleteFido2Registration -> {
                 fido2CompletionManager.completeFido2Registration(event.result)
             }
+
+            is VaultItemListingEvent.Fido2UserVerification -> {
+                if (biometricsManager.isUserVerificationSupported) {
```

Comment:
Since all the other event cases are single lines, I wonder if we can extract this logic into a helper function to keep it nice and clean?